server/views/pages/contacts/manage/contactDetails/manageBasicDetails.njk

```diff
@@ -1,13 +1,9 @@
 
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {% set  formattedDateOfBirth %}{% if contact.dateOfBirth %}{{ contact.dateOfBirth | formatDate }}{% else %}Not provided{% endif %}{% endset %}
-{{ govukSummaryList({
-    card: {
-        title: {
-            text: "Contact details"
-        }
-    },
-    rows: [{
+{% set  estimatedIsOverEighteen %}{% if contact.estimatedIsOverEighteen == 'YES' %}{{ 'Yes' }}{% elseif contact.estimatedIsOverEighteen == 'NO' %}{{ 'No' }}{% else %}{{ 'I don’t know' }}{% endif %}{% endset %}
```

Comment:
So we show "I don't know" if the both date of birth is null and is over 18 is null as well? I think that will be the case for a lot of NOMIS entered contacts (both null). I'm wondering if it should be "Not provided"?

Reply:
for date of birth 'Not provided' is showed

![Screenshot 2024-10-22 at 15 46 03](https://github.com/user-attachments/assets/9a04e884-5077-49b6-9faf-c1fa11039e82)

Reply:
I see from NOMIS, `estimatedIsOverEighteen` may have `null` as value instead of '`YES`', '`NO`' or '`DO_NOT_KNOW`'

Reply:
Yeah, "I don't know" means they answered the question so in the `null` case I think it's misleading.

Reply:
Chris messaged me saying that if both are null `is over 18` will show **'Not provided'**. 

I have amended the `if` condition for `estimatedIsOverEighteen` on the `manageBasicDetails.njk`